web/web-server/src/main/resources/application.yml

```diff
@@ -10,4 +10,11 @@ client:
 
 ldap:
   urls:
-    - ldap://localhost:8389/
\ No newline at end of file
+    - ldap://localhost:8389/
+
+vault:
+  address: http://boom/boom
+  readTimeout: 20
+  openTimeout: 20
+  tokenAuthentication:
+    tokenPath: /var/lib/.tokenz
```

Comment:
Should this config be in here yet?

Reply:
We'd need to know where to lookup the token which will be used to authenticate against a running instance of Vault. Having the path to the file is necessary, and harmless, because if deployed correctly, the file will only be readable by the user running the service.

Reply:
Sorry Kevin, I meant the whole `vault` config in the web `application.yml` - it was commented out in the core yml file - which is why I raised it